server/graphql/index.ts

```diff
@@ -85,6 +85,59 @@ export const rootValue = {
           space
         };
       });
+    },
+    votes: async (
+      parent,
+      { id, first = 10, skip = 0, address, spaces = [], proposal },
+      context,
+      info
+    ) => {
+      const requestedFields = graphqlFields(info);
+      let queryStr = '';
+      const params: any[] = [];
+      if (id) {
+        queryStr += `AND id = ? `;
+        params.push(id);
+      }
+      if (address) {
+        queryStr += `AND address = ? `;
+        params.push(address);
+      }
+      if (spaces.length) {
+        queryStr += `AND space IN (?) `;
+        params.push(spaces);
+      }
+      if (proposal) {
+        queryStr += `AND payload like '%"proposal": "${proposal}"%' `;
```

Comment:
You should always `params` to pass external data so it get encoded otherwise someone could inject SQL queries..